src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/BingAnalyzer.scala

```diff
@@ -1,23 +1,36 @@
 package com.microsoft.partnercatalyst.fortis.spark.analyzer
 
-import java.time.Instant.now
-import java.util.UUID.randomUUID
+import java.text.SimpleDateFormat
+import java.util.TimeZone
+import java.net.URL
 
 import com.github.catalystcode.fortis.spark.streaming.bing.dto.BingPost
 import com.microsoft.partnercatalyst.fortis.spark.transforms.image.ImageAnalyzer
 
 @SerialVersionUID(100L)
 class BingAnalyzer extends Analyzer[BingPost] with Serializable
   with AnalysisDefaults.EnableAll[BingPost] {
+
+  private val defaultFormat: String = "yyyy-MM-dd'T'HH:mm:ss"
```

Comment:
Nit: no need to specify the types explicitly. Also: constants should be `PascalCase`.